systems-at-scale.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: How to Instrument Systems at Scale
+---
+_Before getting onto recommendations on how to instrument your large-scale system with OpenTracing, be sure to read the [Specification overview](/spec)._
+
+<div id="toc"></div>
+
+
+## Spans and Relationships
+
+The two fundamental aspects of implementing OpenTracing across your infrastructure are _Spans_ and the _Relationships_ between those spans:
+
+* **_Spans_** are a logical unit of work in the system that has a start time and a duration. In a trace Spans are the components across the systems exercised on a specified path.
```

Comment:
@yurishkuro you make a good point around consistency. This is something that I have been trying to pay close attention to. In the case of this document the terms are not being re-defined, instead the intent is to use the definitions in context. 

I am actually planning to tackle the spec.md page next and consistencey will be a major focus. This will be part of the effort to make the overall content more organized and approachable. 

I'm not sure that removing the definitions and diagrams from the opening of this document makes it more digestible. In my opinion, the reiteration helps to frame and focus the document. would you be open to leaving it in for v1 and if you still feel that it is redundant after the updates to spec.md we can take it out?